javatests/google/registry/flows/EppFlowInjectionModule.java

```diff
@@ -0,0 +1,46 @@
+package google.registry.flows;
+
+import org.joda.time.DateTime;
+
+import dagger.Module;
+import dagger.Provides;
+import google.registry.flows.EppException.UnimplementedOptionException;
+import google.registry.flows.domain.ExtraDomainValidation;
+import google.registry.model.domain.launch.LaunchCreateExtension;
+import google.registry.model.registry.label.ReservationType;
+
+@Module
+public class EppFlowInjectionModule {
+
+  private final String labelStr = "extra";
+  private final String tldStr = "tld";
+
+  @Provides
+  public ExtraDomainValidation provideExtraDomainValidation() {
+    return new ExtraDomainValidation() {
+      @Override
+      public void validateDomainCreate(String label,
```

Comment:
The formatting in this file is off.  You can either run google-java-format now, or we'll do it before we commit it on our end.